helm-chart-sources/logstream-leader/templates/service-internal.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "logstream-leader.labels" . | nindent 4 }}
   annotations:
-    {{- toYaml .Values.service.annotations  | nindent 4 }}
+    {{- toYaml .Values.service.internalAnnotations  | nindent 4 }}
```

Comment:
This should maintain backwards compatibility such that the `annotations` value is used in both services, or the individual definitions are used.